modules/apps/item-selector/item-selector-test/src/testFunctional/tests/ItemSelector.testcase

```diff
@@ -993,6 +993,65 @@ definition {
 		LexiconEntry.viewNoEntry(rowEntry = "WC WebContent From Site");
 	}
 
+	@description = "The user should see horizontal nav tabs when less than and equal 5 nav items appear on item selector modal."
+	@priority = 4
+	test HorizontalNavItemsShowsWhenNotMoreThan5Items {
+		property test.liferay.virtual.instance = "false";
+
+		HeadlessSite.addSite(siteName = "Site Name");
+
+		JSONLayout.addPublicLayout(
+			groupName = "Site Name",
+			layoutName = "Content Page Name",
+			type = "content");
+
+		ApplicationsMenu.gotoPortlet(
+			category = "System",
+			panel = "Control Panel",
+			portlet = "App Manager");
+
+		AppManager.viewApp(
+			appName = "Knowledge Base",
+			appStatus = "Active");
+
+		AppManager.deactivateApp(appName = "Knowledge Base Web");
```

Comment:
Can you check if there is a way to exclude the app from deploying or deactivate it from the backend?

Reply:
Use config file to blacklist modules.